### DISCUSSION TITLE
v-slider: 2-way data binding in own component

### DISCUSSION THREAD

**Discussion by Kugeleis** [vuetifyjs/vuetify#16320] [Q&A]
I try to put my vuetify3 slider in a separate vue component. But [following the example](https://vuejs.org/guide/components/events.html#usage-with-v-model) I have no "value" in slider and using v-model gives an error. What is the correct serup for a v-slider as a separate vue-component?

**Comment by yuwu9145** [comment]
Could you provide a reproduction link which shows the exact error?
